test/unit/gateways/decidir_test.rb

```diff
@@ -278,6 +278,53 @@ def test_payment_method_id_with_naranja
     assert_equal 24, post[:payment_method_id]
   end
 
+  def test_payment_method_id_with_visa_debit
+    visa_debit_card = credit_card('4517721004856075')
+    debit_options = @options.merge(payment_method_id: 31)
```

Comment:
I wonder if we should we write a separate test for passing a visa with `debit: true` for completeness' sake, but do feel like the code path is adequately tested without it. Up to you!

Reply:
Shoot! That was definitely supposed to be `debit: true`! 

While I was working on adding this, I discovered that the existing GSF for `payment_method_id` didn't work, so I was troubleshooting that and forgot to change it back. Thanks for pointing that out!